app/views/pageflow/social_share/_entry_meta_tags.html.erb

```diff
@@ -2,10 +2,9 @@
 
 <%= social_share_entry_image_tags(entry) %>
 
-<%= tag :meta, :property => "og:title", :content => "#{entry.title} - #{entry.theming.cname_domain}" %>
+<%= tag :meta, :property => "og:title", :content => pretty_entry_title(entry) %>
 <%= tag :meta, :property => "og:description", :name => "description", :content => social_share_entry_description(entry) %>
 <%= tag :meta, :property => "og:url", :content => pretty_entry_url(entry) %>
-<%= tag :meta, :property => "og:site_name", :content => entry.theming.cname_domain %>
```

Comment:
In my opinion, it does not follow the [OpenGraph site_name spec](http://ogp.me):

> If your object is part of a larger web site, the name which should be displayed for the overall site. e.g., "IMDb".

But it's not a deal breaker for me to keep it in. :)